sass/badge_only.sass

```diff
@@ -89,11 +89,9 @@
         display: block
 
 .injected-customised
-  height: 100px
   .rst-versions
     width: 100%
     font-size: .8em
-    height: 100px
     pointer-events: initial
```

Comment:
C'era un motivo per questi `height`, e mi sa che riguardava il posizionamento quando si scrolla in basso nella pagina, quando quel layerino da `fixed` si appoggia al footer (l'elemento saltellava). Merita un check aggiuntivo, se non l'hai già fatto. Se noti particolari problemi, LGTM.

Reply:
Ho verificato il bug che rilevi. Ripristino l'impostazione precedente.